tests/integration/chunkstore/test_chunkstore.py

```diff
@@ -465,7 +465,7 @@ def helper(chunkstore_lib, chunk_size, name, df, append):
 
         chunkstore_lib.update(name, append)
 
-        if chunk_size is not 'A':
+        if chunk_size != 'A':
```

Comment:
is this performant?

Reply:
it's a warning from the linter, should use `!=` for comparisons instead of `is not`